### ISSUE TITLE
After running these command  "python Tensorflow/models/research/object_detection/model_main_tf2.py --model_dir=Tensorflow/workspace/models/my_ssd_mobnet --pipeline_config_path=Tensorflow/workspace/models/my_ssd_mobnet/pipeline.config --num_train_steps=5000" not showing anything .

### ISSUE THREAD

**Issue by Amit998** [nicknochnack/RealTimeObjectDetection#5]
D:\study\python\AI\Deep Learning\convolutional neural network(CNN)\projects\Real Time Face Mask Detection with Tensorflow and Python>python Tensorflow/models/research/object_detection/model_main_tf2.py --model_dir=Tensorflow/workspace/models/my_ssd_mobnet --pipeline_config_path=Tensorflow/workspace/models/my_ssd_mobnet/pipeline.config --num_train_steps=5000
2021-03-12 20:14:31.022628: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cudart64_110.dll
C:\Users\damit\AppData\Local\Programs\Python\Python37\lib\site-packages\tensorflow_addons\utils\ensure_tf_install.py:43: UserWarning: You are currently using a nightly version of TensorFlow (2.5.0-dev20210311).
TensorFlow Addons offers no support for the nightly versions of TensorFlow. Some things might work, some other might not.
If you encounter a bug, do not file an issue on GitHub.
  UserWarning,
2021-03-12 20:14:34.805533: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library nvcuda.dll
2021-03-12 20:14:34.838992: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1779] Found device 0 with properties:
pciBusID: 0000:01:00.0 name: GeForce GTX 1650 computeCapability: 7.5
coreClock: 1.56GHz coreCount: 16 deviceMemorySize: 4.00GiB deviceMemoryBandwidth: 119.24GiB/s
2021-03-12 20:14:34.839139: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cudart64_110.dll
2021-03-12 20:14:34.863061: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cublas64_11.dll
2021-03-12 20:14:34.863191: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cublasLt64_11.dll
2021-03-12 20:14:34.880273: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cufft64_10.dll 
2021-03-12 20:14:34.884287: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library curand64_10.dll
2021-03-12 20:14:34.930953: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cusolver64_10.dll
2021-03-12 20:14:34.945747: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cusparse64_11.dll
2021-03-12 20:14:34.946548: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cudnn64_8.dll
2021-03-12 20:14:34.946722: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1917] Adding visible gpu devices: 0
2021-03-12 20:14:34.947126: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2021-03-12 20:14:34.948525: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1779] Found device 0 with properties:
pciBusID: 0000:01:00.0 name: GeForce GTX 1650 computeCapability: 7.5
coreClock: 1.56GHz coreCount: 16 deviceMemorySize: 4.00GiB deviceMemoryBandwidth: 119.24GiB/s
2021-03-12 20:14:34.949145: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1917] Adding visible gpu devices: 0
2021-03-12 20:14:35.473383: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1304] Device interconnect StreamExecutor with strength 1 edge matrix:
2021-03-12 20:14:35.473677: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1310]      0 
2021-03-12 20:14:35.474115: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1323] 0:   N
2021-03-12 20:14:35.474626: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1464] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 2167 MB memory) -> physical GPU (device: 
0, name: GeForce GTX 1650, pci bus id: 0000:01:00.0, compute capability: 7.5)
INFO:tensorflow:Using MirroredStrategy with devices ('/job:localhost/replica:0/task:0/device:GPU:0',)
I0312 20:14:35.478294 16912 mirrored_strategy.py:363] Using MirroredStrategy with devices ('/job:localhost/replica:0/task:0/device:GPU:0',)
INFO:tensorflow:Maybe overwriting train_steps: 5000
I0312 20:14:35.486292 16912 config_util.py:552] Maybe overwriting train_steps: 5000
INFO:tensorflow:Maybe overwriting use_bfloat16: False
I0312 20:14:35.487290 16912 config_util.py:552] Maybe overwriting use_bfloat16: False
Windows fatal exception: stack overflow

Current thread 0x00004210 (most recent call first):
  File "C:\Users\damit\AppData\Local\Programs\Python\Python37\lib\site-packages\tensorflow\python\ops\gen_math_ops.py", line 6240 in mul
  File "C:\Users\damit\AppData\Local\Programs\Python\Python37\lib\site-packages\tensorflow\python\ops\math_ops.py", line 530 in multiply
  File "C:\Users\damit\AppData\Local\Programs\Python\Python37\lib\site-packages\tensorflow\python\util\dispatch.py", line 206 in wrapper
  File "C:\Users\damit\AppData\Local\Programs\Python\Python37\lib\site-packages\tensorflow\python\ops\math_ops.py", line 1537 in _mul_dispatch
  File "C:\Users\damit\AppData\Local\Programs\Python\Python37\lib\site-packages\tensorflow\python\ops\math_ops.py", line 1236 in r_binary_op_wrapper
  File "C:\Users\damit\AppData\Roaming\Python\Python37\site-packages\object_detection\models\ssd_mobilenet_v2_fpn_keras_feature_extractor.py", line 203 in preprocess
  File "C:\Users\damit\AppData\Roaming\Python\Python37\site-packages\object_detection\meta_architectures\ssd_meta_arch.py", line 482 in preprocess
  File "C:\Users\damit\AppData\Roaming\Python\Python37\site-packages\object_detection\model_lib_v2.py", line 524 in train_loop
  File "Tensorflow/models/research/object_detection/model_main_tf2.py", line 110 in main
  File "C:\Users\damit\AppData\Local\Programs\Python\Python37\lib\site-packages\absl\app.py", line 251 in _run_main
  File "C:\Users\damit\AppData\Local\Programs\Python\Python37\lib\site-packages\absl\app.py", line 300 in run
  File "C:\Users\damit\AppData\Local\Programs\Python\Python37\lib\site-packages\tensorflow\python\platform\app.py", line 40 in run
  File "Tensorflow/models/research/object_detection/model_main_tf2.py", line 113 in <module>

D:\study\python\AI\Deep Learning\convolutional neural network(CNN)\projects\Real Time Face Mask Detection with Tensorflow and Python>

**Comment by Amit998** [comment]
don't understand what is the issue.and these logs
